nixos/modules/security/acme.nix

```diff
@@ -111,6 +96,31 @@ let
           own server roots if needed.
         '';
       };
+
+      dnsProvider = mkOption {
+        type = types.nullOr types.str;
+        default = null;
+        example = "route53";
+        description = "DNS Challenge provider";
+      };
+
+      credentialsFile = mkOption {
+        type = types.str;
+        description = ''
+          File containing DNS provider credentials passed as environment variables.
+          See https://go-acme.github.io/lego/dns/ for more information.
```

Comment:
Maybe make the format clearer by mentioning that this is passed into the EnvironmentFile option of the systemd unit.

Reply:
How does it sound now?

Reply:
Great, thank you!